git_machete/client.py

```diff
@@ -1533,18 +1535,20 @@ def handle_untracked_branch(self, new_remote: str, branch: str, is_called_from_t
         other_remote_choice = "o[ther-remote]" if can_pick_other_remote else ""
         remote_branch = f"{new_remote}/{branch}"
         if not self.__git.get_commit_sha_by_revision(remote_branch, prefix="refs/remotes/"):
-            choices = get_pretty_choices(*('y', 'N', 'q', 'yq', other_remote_choice) if is_called_from_traverse else ('y', 'q', other_remote_choice))
+            choices = get_pretty_choices(*('y', 'N', 'q', 'yq', other_remote_choice) if is_called_from_traverse else ('Y', 'q', other_remote_choice))
```

Comment:
Hmmm for consistency with all other places I'd rather make `Q` (i.e. **refraining** from action, rather than proceeding with the action) the default choice